RAW/WebService.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  WebService.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import Foundation
+
+class WebService: NSObject {
+    
```

Comment:
How about we make the service a protocol and then make a concrete type that conforms to it? It will help us in testing.

Reply:
Yes